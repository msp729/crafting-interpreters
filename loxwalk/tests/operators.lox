print 1 / 2;
print "a" + true;
print nil + "";
print false;
