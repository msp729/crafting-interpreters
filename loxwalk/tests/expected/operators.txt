0.5
atrue
nil
false
